exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  } else if (mark < 80 && mark > 60) {
```

Comment:
Here in your both else if blocks in line 14 and 16 you wouldn't need to check that the mark is smaller than 80 as it was already checked. Something like: else if (mark > 60) return "B". This is something I just learned from Nigel and is very helpful to know